### PR TITLE
build: use whiskers

### PR DESCRIPTION
**Pull request by uncenter**
Adds [Whiskers](https://github.com/catppuccin/toolbox/tree/main/whiskers), Catppuccin's internal templating tool, to build the themes. Instead of the Lua script, edit the `diohub.tera` file and then run `just build` to build the README with the links for each flavor. You will need Whiskers (linked above) and *optionally* [Just](https://github.com/casey/just) installed.

While converting this I noticed that all of the links for each different accent had the same link with no difference as you would expect from supposedly having a different accent. As such, this new template only provides a link *per flavor*, since that was in essence what was happening before.

Screenshot 2024-05-26 at 18 36 39 (Arc)

### CODE REVIEW

**Comment by asumbek** [comment]
Thank you for the pull request, I was planning to use Whisker instead of Lua script, but I don't have any interest or time working on it. And about the links, I'm 100% sure it should generate different link with only difference in `accent` key, but thanks for noticing it for me.

**Comment by asumbek** [comment]
I just rechecked the README links, it should contain `accent` key for different accents, weird.

File: diohub.tera [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+> [!NOTE]
+>
+> As of current pre-release version `v1.0.0-dev.1`, this theme would not work for that version and beyond. Changes are currently made for this version will eventually be pushed to the upstream.
+
+1. Ensure [DioHub](https://github.com/NamanShergill/diohub) is installed.
+2. Open the link below:
+{%- for _, flavor in flavors %}
+{%- set palette = flavor.colors %}
+- [{{ flavor.emoji }} {{ flavor.name }}](https://theme.felix.diohub?format_ver=0&elementsOnColors=ff{{ palette.subtext1.hex }}&faded2=ff{{ palette.overlay1.hex }}&green=ff{{ palette.green.hex }}&baseElements=ff{{ palette.subtext0.hex }}&faded3=ff{{ palette.overlay2.hex }}&secondary=ff{{ palette.crust.hex }}&primary=ff{{ palette.base.hex }}&faded1=ff{{ palette.overlay0.hex }}&red=ff{{ palette.red.hex }})
+{%- endfor %}
```
Comment (asumbek) [inline diohub.tera]: Can you add here so it would include `&accent=<hex>` where `<hex>` is the accent color?

**Comment by uncenter** [comment]
> I just rechecked the README links, it should contain `accent` key for different accents, weird.

I've updated the template to include your changes.

**Review by asumbek — APPROVED** [review]
Thank you

**Pull request merged**